orchagent/muxorch.cpp

```diff
@@ -566,6 +597,29 @@ void MuxCable::updateNeighbor(NextHopKey nh, bool add)
     else if (mux_name_ == mux_orch_->getNexthopMuxName(nh))
     {
         mux_orch_->removeNexthop(nh);
+        nbr_handler_->removeNeighborRoutes(nh);
```

Comment:
This part is not clear. If a nh is removed for some reason, why remove the Route itself?